docs/changes.rst

```diff
@@ -3,6 +3,11 @@
 Changelog
 =========
 
+Changes in 0.11.3.dev0
+----------------------
+
+- Restored support for Django < 1.8 & Python 2.6
```

Comment:
Should maybe say `...support for Django 1.4, 1.5, 1.6, and 1.7 ...` just to be more clear.. otherwise I think someone using one of these versions would still have to visit the code to see.